camp/challenges.md

```diff
@@ -26,7 +26,7 @@ The table below highlights various challanges that coder members will be assigne
 |Input Hardware | Output Hardware | Expected Outcome | Target Owner | Owner |
 |--|--|--|--|--|
 | Mag sensor | bag motor on/off in a single direction | Every time a beam break sensor is broken turn motor on, otherwise turn off motor | 2nd year | ??  |
-| Tof/Distance sensor | Small neo | As something comes into view of the sensor closer and closer motor speed increases.  When nothing is in view the motor is off | 2nd year | ??  |
+| Tof/Distance sensor | Small neo | As something comes into view of the sensor closer and closer motor speed increases.  When nothing is in view the motor is off | 2nd year | Isaac Bess  |
```

Comment:
There is a merge conflict here w/ main branch - don't forget to pull the latest & resolve the conflict